microsite/data/plugins/splunk-on-call.yaml

```diff
@@ -0,0 +1,14 @@
+---
+title: Splunk On-Call
+author: Spotify
```

Comment:
You don't have to put Spotify here you know either, you can use your own handle if you wanted? Feel free to raise a PR after if you like. I'm gonna merge this now :)